### PR TITLE
Added emailIssueToc plugin compatibility for 3.2+ and 3.3

### PR DESCRIPTION
**Pull request by wopsononock**

### CODE REVIEW

**Review by ctgraham — APPROVED** [review]

File: plugins.xml [hunk 1]
```diff
@@ -4531,6 +4531,101 @@
...
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
+				<version>3.3.0.9</version>
+				<version>3.3.0.10</version>
+			</compatibility>
+			<compatibility application="omp">
```
Comment (ctgraham) [inline plugins.xml:4549]: Please rescind the compatibility statement with OMP and OPS.  This plugin is specific to Journal Issue Table of Contents.

**Review by ctgraham — CHANGES_REQUESTED** [review]

File: plugins.xml [hunk 2]
```diff
@@ -4531,6 +4531,75 @@
...
+		<release date="2022-04-29" version="1.0.2.2" md5="4456324a6deb504831e11c328bd49fa3">
+			<package>https://github.com/ulsdevteam/pkp-emailIssueToc/releases/download/v1.0.2-2/emailIssueToc-1.0.2-2.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.2.1.0</version>
+				<version>3.2.1.1</version>
+				<version>3.2.1.2</version>
+				<version>3.2.1.3</version>
+				<version>3.2.1.4</version>
+			</compatibility>
+			<compatibility application="ojs2">
```
Comment (ctgraham) [inline plugins.xml:4561]: This compatibility statement should be merged with the above, for just one "ojs2" application statement.

File: plugins.xml [hunk 3]
```diff
@@ -4531,6 +4531,75 @@
...
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
+				<version>3.3.0.9</version>
+				<version>3.3.0.10</version>
+			</compatibility>
+			<compatibility application="omp">
```
Comment (ctgraham) [inline plugins.xml:4574]: OMP and OPS compatibility should be removed here as well.

**Review by ctgraham — CHANGES_REQUESTED** [review]

**Review by ctgraham — APPROVED** [review]

**Comment by asmecher** [comment]
Merged, thanks!

**Pull request merged**